src/backend/executor/cypher_create.c

```diff
@@ -613,7 +629,7 @@ static Datum create_vertex(cypher_create_custom_scan_state *css,
          */
         if (!SAFE_TO_SKIP_EXISTENCE_CHECK(node->flags))
         {
-            if (!entity_exists(estate, css->graph_oid, DATUM_GET_GRAPHID(id)))
+            if (!entity_exists(estate, css->graph_oid, DATUM_GET_GRAPHID(id), label_id))
```

Comment:
Replace `DATUM_GET_GRAPHID` with something equivalent for bigint. Because we are dropping that `graphid` type.